src/webrtc.ts

```diff
@@ -197,9 +273,12 @@ export class WebrtcExtension extends EventEmitter {
       'endOfCandidates'
     ];
     for (const e of eventsToProxy) {
-      this.client._stanzaio.jingle.on(e, (session: GenesysCloudMediaSession, ...data: any) => {
-        session.emit(e as any, ...data);
-      });
+      this.client._stanzaio.jingle.on(
```

Comment:
Can you change your prettier plugin or what ever this is so it doesn't affect "unchanged" code? Otherwise this will probably get pretty messing with bigger projects.

Reply:
Yeah I'm going to just disable it for now, not sure what the deal is with its config right now.